packages/openapi3/test/openapi-output.test.ts

```diff
@@ -249,6 +249,23 @@ describe("openapi3: extension decorator", () => {
     strictEqual(oapi.components.parameters.PetId["x-parameter-extension"], "foobaz");
   });
 
+  it("adds an extension to a namespace", async () => {
+    const oapi = await openApiFor(`
+      @extension("x-model-extension", "foobar")
+      @route("/createSong")
+      namespace Suno {
+        @get op createSong(
+          /** The requested song description, eg. \`a country song about Thanksgiving\` */
+          @query topic: string
+          ): string;
+      }      
+    `);
+
+    ok(oapi);
+    const extensions = oapi.getExtensions();
```

Comment:
Does this exist?

Reply:
No it doesn't. I've gotten rid of it and updated the test as per your feedback